Exercise_4/docs/part-1.md

```diff
@@ -0,0 +1,330 @@
+
+Managed by the Regenstrief Institute, LOINC has been identified by the Health Level 7 (HL7) Standards Development Organization as a preferred diagnostic report code set for laboratory test identification in transactions between health care facilities, laboratories, laboratory testing devices and public health authorities.
+
+See here: [https://www.hl7.org/fhir/diagnosticreport-definitions.html#DiagnosticReport.code](https://www.hl7.org/fhir/diagnosticreport-definitions.html#DiagnosticReport.code)
+
+
+LOINC codes are dvidied into Test Order and Test Result codes.
+
+??? abstract "Test Order Code Structure and Examples"
+    Test Order codes are 5 digit numeric codes appended with a dash and 6th number. Form: XXXXX-X
+
+    e.g.,
+        As an example, one of the more popular codes is: `2160-0` for `Creatinine [Mass/volume] in Serum or Plasma`
+        Source:  [https://loinc.org/2160-0/](https://loinc.org/2160-0/)
+
+    Additional attributes associated with LOINC test codes are:
+
+    * Test ID
+    * Test Order Name
+    * Order LOINC Code
+    * LOINC Attributes
+    * Method name
+
+
+    e.g.,
+    An example can be found here: [https://loinc.org/2160-0/](https://loinc.org/2160-0/) and [https://loinc.org/kb/users-guide/introduction/](https://loinc.org/kb/users-guide/introduction/)
+
+
+    | Test ID | Test Order Name | Order LOINC Code | LOINC Attributes | Method Name |
+    |---------|-----------------|------------------|------------------|-------------|
+    | 10AIH  | Factor X Inhibitor Profile, Professional Interpretation | 69049-5 | Coagulation specialist review:Imp:Pt:XXX:Nar: | Only orderable as a reflex. For more information see 10INE / Factor X Inhibitor Evaluation, Plasma. Medical Interpretation |
+
+??? abstract "Test Result Code Structure and Examples"
+    LOINC Test Result codes are also 5 digit numeric codes appended with a dash and 6th number. Form: XXXXX-X
+
+    Additional attributes associated with LOINC Test Result codes are:
+
+    * LOINC Values Updated
+    * Test ID
+    * Test Order Name
+    * Result ID
+    * Result LOINC Code
+    * LOINC Attributes
+    * Method Name
+
+    | LOINC Values Updated | Test ID | Test Order Name | Result ID | Test Result Name| Result LOINC Code | LOINC Attributes | Method Name |
+    |----------------------|---------|-----------------|-----------|-----------------|-------------------|------------------|-------------|
+    | 2022-03-11 | CNSA | Central Nervous System Consultation, Autopsy, Varies | 71070 | Interpretation | 18743-5 | Autopsy report:Find:Pt:{Setting}:Doc:{Role} | Includes hermatoxylin and eosin stains. Additional special stains or procedures necessary for diagnosis will be billed separately. |
+
+!!! check "For more information on LOINC codes"
+    Additonal infomration on the **LOINC coding terminology** is available here: [https://loinc.org]( https://loinc.org)
```

Comment:
Would be nice to have a standardized way of giving links to additional information in all the sections.

```suggestion
!!! check "For more information on LOINC codes"
    Additional information on the **LOINC coding terminology** is available here: [https://loinc.org]( https://loinc.org)
```